README.md

```diff
@@ -213,6 +213,20 @@ resources in the `"included"` member when the resource names are included in the
   render @posts, include: 'author,comments,comments.author'
 ```
 
+In addition, two types of wildcards may be used. `*` includes one level of associations, and `**` includes all recursively.
+
+```ruby
+  render @posts, include: '**' # or '*' for a single layer
```

Comment:
Maybe it would be nice to include an example mixing both paths and wildcards, as wildcards can be used pretty much anywhere (stuff like `author.comments.*.*` (you could even do stuff such as `author.comments.*.*.posts`, although I can't really think of a use case which such homogeneous resources that it would make sense, maybe some polymorphic association stuff), and `author.comments.**`).